internal/uvm/create_wcow.go

```diff
@@ -319,6 +326,8 @@ func CreateWCOW(ctx context.Context, opts *OptionsWCOW) (_ *UtilityVM, err error
 		return nil, fmt.Errorf("error while creating the compute system: %s", err)
 	}
 
+	uvm.cpuGroupID = opts.CPUGroupID
```

Comment:
Do we actually use this value anywhere?

Reply:
It's used on close to move the cpugroup to the null ID, which doesn't seem necessary really. Anytime we close the vm we're generally tearing it down anyways.

Reply:
I left in the assignment of the value for whenever we might use reassignments, even though the runtime add/remove functions are no longer used.

Reply:
Can we just remove this field? It looks like we don't really need it in `ReleaseCPUGroup`.

Reply:
Your comment didn't display earlier. I'm not sure we need to actually track this, but it's not a big deal.

Reply:
As in my comments not showing up on github? 😮 Anyways, I'd prefer to leave it for now even if unused, this was mostly code that was already here so I'm not sure what was taken into account when adding this. It looks like it's used in Release to check if we have a cpugroup assigned in the first place so removal won't whine.

Reply:
This is going to get changed a bit when the abstractions for all these operations gets in anyways so we can reevaluate then also.

Reply:
That's fine with me

Reply:
IMO we can just take it out. It was really only used to decide if we should add the VM to the null group on release and as an optimization so we wouldn't try to assign a VM to a group when it was already in that group, but I don't think that really matters.

Reply:
Let's just axe it. done